starter-code/src/viking.js

```diff
@@ -1,11 +1,104 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+}
+
+Soldier.prototype.attack=function(){
+    return this.strength;
+}
+
+Soldier.prototype.receiveDamage=function(damage){
+    this.health -= damage;
+}
 
 // Viking
-function Viking() {}
+Viking.prototype=Object.create(Soldier.prototype);
+
+function Viking (name,health,strength){
+    this.name=name;
+    this.health=health;
+    this.strength=strength;
+}
+
```

Comment:
Here you basically recreated the Viking from scratch, while the idea was to import characteristics from the Soldier constructor.
In this case it's not a big deal, because the Soldier is pretty simple, but sometimes you'll face super complex constructor functions that you don't want to be copying line by line ;)

The idea was to use the .call method, as follows:

```javascript
function Viking( name, health, strength ) {
    Soldier.call( this, health, strength );
    this.name = name;
};
```